src/parser.rs

```diff
@@ -221,6 +231,12 @@ impl<'a, 'b> Parser<'a, 'b> {
     }
 
     pub(crate) fn consume_plain_identifier(&mut self) -> Result<Identifier<'a>, ParseError> {
+        if self.options.dialect.is_postgresql() {
+            if let Token::DoubleQuotedString(v) = &self.token {
+                let v = *v;
+                return Ok(Identifier::new(v, self.consume()));
+            }
```

Comment:
I think it would be prettier if this was
```rust
Token::DoubleQuotedString(v) if self.options.dialect.is_postgresql() =>
```
in the matchstatement